### PR TITLE
Add Logging SDK features to spec compliance matrix

### PR DESCRIPTION
**Pull request by tigrannajaryan**
We will start tracking logging feature implementations.

Once this PR is merged we will need existing Java, Python and .Net
implementations to populate the matrix.

### CODE REVIEW

**Review by djaglowski — APPROVED** [review]

**Review by reyang — APPROVED** [review]

File: spec-compliance-matrix.md [hunk 1]
```diff
@@ -208,6 +208,27 @@ Disclaimer: this list of features is still a work in progress, please refer to t
 | The metrics SDK provides an `ExemplarFilter` interface or extension point.                                                                                                   | X        |    |      |    |    -   |      |        |     |      |     |   -  |       |
 | An `ExemplarFilter` has access to the measurement value, attributes, `Context` and timestamp.                                                                                | X        |    |      |    |    -   |      |        |     |      |     |   -  |       |
 
+## Logs
+
+**Status**: [Experimental](./specification/document-status.md)
```
Comment (reyang) [inline spec-compliance-matrix.md:213]: Do we need "Status" here? This is not a spec. (it seems we have it for Metrics matrix, which seems to be a bug to me)

Comment (reyang) [reply spec-compliance-matrix.md:213]: #2474

Comment (tigrannajaryan) [reply spec-compliance-matrix.md:213]: Yep, copied from Metrics section. Don't know if we need it, happy to remove.

File: spec-compliance-matrix.md [hunk 2]
```diff
@@ -208,6 +208,27 @@ Disclaimer: this list of features is still a work in progress, please refer to t
...
+
+Disclaimer: this list of features is still a work in progress, please refer to the specification if in any doubt.
+
+| Feature                                                     | Optional | Go | Java | JS | Python | Ruby | Erlang | PHP | Rust | C++ | .NET | Swift |
+|-------------------------------------------------------------|----------|----|------|----|--------|------|--------|-----|------|-----|------|-------|
+| **[Logging SDK](specification/logs/logging-library-sdk.md)**| Optional | Go | Java | JS | Python | Ruby | Erlang | PHP | Rust | C++ | .NET | Swift |
+| Get LogEmitter                                              |          |    |      |    |        |      |        |     |      |     |      |       |
```
Comment (arminru) [inline spec-compliance-matrix.md:220]: What about LogEmitterProvider.Shutdown and LogEmitterProvider.ForceFlush?

**Review by arminru — APPROVED** [review]

File: spec-compliance-matrix.md (hunk 1 above)
Comment (tigrannajaryan) [reply spec-compliance-matrix.md]: Removed.

File: spec-compliance-matrix.md (hunk 2 above)
Comment (tigrannajaryan) [reply spec-compliance-matrix.md:218]: Added.

**Pull request merged**